### PR TITLE
DOC: I write about set_true_EN_PASSANT

### LINKED ISSUE #35: unclear function set_true_en_passant
I am going to create a comment for this function

### PR DESCRIPTION
**Pull request by Ahmedniz**
Closed  #35

### CODE REVIEW

File: Chess_Board/src/board.py [hunk 1]
```diff
@@ -67,8 +67,10 @@ def check_promotion(self, piece, final):
     def castling(self, initial, final):
         return abs(initial.col - final.col) == 2
 
+    # this function is suppose  to set all piece to false en_passant except pawn  
     def set_true_en_passant(self, piece):
         
+
```
Comment (Salman-18) [inline Chess_Board/src/board.py:73]: @Ahmedniz  why did you add this blank line?

Comment (Ahmedniz) [reply Chess_Board/src/board.py:73]: @Salman-18 Ok let me fix that

**Comment by Salman-18** [comment]
LGTM!
